exec/java-exec/src/main/java/org/apache/drill/exec/record/RecordBatchSizer.java

```diff
@@ -108,6 +108,12 @@
     public final float estElementCountPerArray;
     public final boolean isVariableWidth;
 
+    public Map<String, ColumnSize> childColumnSizes = CaseInsensitiveMap.newHashMap();
```

Comment:
Nit: `children` might be shorter and simpler. It is clear that the children are sizes from the member type.